K2Bridge/Factories/BucketFactory.cs

```diff
@@ -84,5 +85,60 @@ public static TermsBucket CreateTermsBucketFromDataRow(DataRow row)
 
             return tb;
         }
+
+        /// <summary>
+        /// Create a new <see cref="RangeBucket" from a given <see cref="DataRow"/>/>.
+        /// </summary>
+        /// <param name="row">The row to be transformed to bucket.</param>
+        /// <returns>A new TermsBucket.</returns>
+        public static RangeBucket CreateRangeBucketFromDataRow(DataRow row)
+        {
+            Ensure.IsNotNull(row, nameof(row));
+
+            var range = Convert.ToString(row[(int)BucketColumnNames.Timestamp]);
```

Comment:
Actually since the constant is used in several places, I renamed it to `SummarizeByColumn` to reflect that it is really the indx of the column created by `summarize ... by`.